basic-algorithms-&-datastructures/index070.js

```diff
@@ -0,0 +1,33 @@
+const newsFeed = {
+  page: "front_page",
+    articles: [{
+      headline: "Programming: Expectations vs. Reality",
+      image: [],
+      date_published: "2019-01-05T08:00:00+08:00",
+      date_modified: "2019-01-05T09:20:00+08:00",
+      author: {
+        username: "PeoplePerson",
+        name: "Pat Comb"
+      },
+      likes: 10571,
+      dislikes: 14,
+      shares: 54,
+      comments: 461,
+      section: {
+        name: "Programming",
+        subscribers: 18101
+      },
+      url: "http://www.binary-bugle.com/programming-expectations-vs-reality"
+    },{},{},{},{},{}
+  ]}
+
+/*
+To access nested objects we can also use bracket notation and a numerical value to select with object as index. *Note computers count values starting at zero.
```

Comment:
Building off of my last comment, this could possibly be reworked into a "new username" function later.